src/STEMMUS_SCOPE.m

```diff
@@ -369,6 +369,9 @@
         if isnan(GroundwaterSettings.tempBotm)
             GroundwaterSettings.tempBotm = SoilVariables.TT(GroundwaterSettings.indxBotmLayer);
         end
+
+        % Initialize groundwater recharge temperature
+        gwfluxes.rechargeTemp = 0;
```

Comment:
```suggestion
```

Reply:
the variable `gwfluxes.rechargeTemp` is not getting updated and not used to calculate other variables. What is the reason to initialize it with zero?

Reply:
> the variable `gwfluxes.rechargeTemp` is not getting updated and not used to calculate other variables. What is the reason to initialize it with zero?

True, there is no need. I was just testing something. Its removed now